index.js

```diff
@@ -256,6 +256,10 @@
     };
   };
 
+  var compose3 = function(f, g, x) {
+    return f(g(x));
+  };
+
```

Comment:
I wasn't sure how to deal with aliases in Sanctuary Def. Just going `S.compose = S.I` would give `compose` the name of `I`, going `def('compose', ..., S.I)` would do double type-checking I think. So this is what I ended up doing.